books/RayTracingTheRestOfYourLife.html

```diff
@@ -1269,109 +1287,148 @@
 Light Scattering
 ====================================================================================================
 
-In this chapter we won't actually program anything. We will set up for a big lighting change in the
-next chapter.
+In this chapter we won't actually program anything. We'll just be setting up for a big lighting
+change in the next chapter. Our ray tracing program from the first two books scatters a ray when it
+interacts with a surface or a volume. Ray scattering is the most commonly used model for simulating
+light propagation through a scene. This can naturally be modeled probabilistically. There are many
+things to consider when modeling the probabilistic scattering of rays.
 
 
 Albedo
 -------
-Our program from the last books already scatters rays from a surface or volume. This is the commonly
-used model for light interacting with a surface. One natural way to model this is with probability.
 First, is the light absorbed?
 
-Probability of light scattering: $A$
+Probability of light being scattered: $A$
 
 Probability of light being absorbed: $1-A$
 
-Here $A$ stands for _albedo_ (latin for _whiteness_). Albedo is a precise technical term in some
-disciplines, but in all cases it is used to define some form of _fractional reflectance_. This
-_fractional reflectance_ (or albedo) will vary with color and (as we implemented for our glass in
-book one) can vary with incident direction.
+Where here $A$ stands for _albedo_, which is latin for _whiteness_. Albedo is a precise technical
+term in some disciplines, but in all cases it is used to define some form of
+_fractional reflectance_. This _fractional reflectance_ (or albedo) will vary with color and (as we
+implemented for our glass in book one) can also vary with incident direction (the direction of the
+incoming ray). It can help to stop and remember that when we simulate light propagation all we're
+doing is simulating the movement of photons through a space. If you remember your high school
+Physics then you should recall that every photon has a unique energy and wavelength associated by
+the Planck constant:
 
+    $$ E = \frac{hc}{\lambda} $$
+
+Each individual photon has a _tiny_ amount of energy, but when you add enough of them up you get all
+of the illumination in your program. The absorption or scattering of a photon with a surface or a
+volume (or really anything that a photon can interact with) is probabilistically determined by the
+albedo of the object. Albedo can depend on color because some objects are more likely to absorb
+some wavelengths.
 
 Scattering
 -----------
 In most physically based renderers, we would use a set of wavelengths for the light color rather
 than RGB. We can extend our intuition by thinking of R, G, and B as specific algebraic mixtures of
```

Comment:
rather than just three specific wavelengths (red, green and blue).

Reply:
Outside of Rec2020:
https://en.wikipedia.org/wiki/Rec._2020

RGB are mixtures of wavelengths.

Reply:
But point taken